Library/Homebrew/download_strategy.rb

```diff
@@ -312,6 +329,10 @@ def _fetch
     curl @url, "-C", downloaded_size, "-o", temporary_path
   end
 
+  def does_redirect
```

Comment:
I'm not overly familiar with ` open-uri `, and given we're already shelling out plenty in ` fetch ` I didn't think one more was particularly gratuitous. Also somewhat worry about Ruby not being great at backwards compatibility - Will ` open-uri ` work out of the box right back to 10.4/10.5's Ruby? I'd prefer not to create more work for Misty if not.